ext/intl/tests/bug62017.phpt

```diff
@@ -5,19 +5,22 @@ intl
 --FILE--
 <?php
 ini_set('intl.error_level', E_WARNING);
-var_dump(
+try {
     datefmt_create('', IntlDateFormatter::NONE, IntlDateFormatter::NONE, "\xFF",
-        IntlDateFormatter::GREGORIAN, 'a'));
+        IntlDateFormatter::GREGORIAN, 'a');
+} catch (IntlException $e) {
+    echo PHP_EOL."Exception: " . $e->getMessage() . " in " . $e->getFile() . " on line " . $e->getLine() . PHP_EOL;
```

Comment:
Might be sufficient to only printf `$e->getMessage()`, but that should be done consistently for this test case. Leaving as is, is also fine for me.